build/azure-pipelines.yml

```diff
@@ -110,10 +110,7 @@ stages:
                   }
                 }
 
-                foreach($csproj in Get-ChildItem -Recurse -Filter *.csproj)
-                {
-                  dotnet pack $csproj --configuration $(buildConfiguration) --no-build --output $(Build.ArtifactStagingDirectory)/nupkg
-                }
+                dotnet pack $(solution) --configuration $(buildConfiguration) --no-build --property:PackageOutputPath=$(Build.ArtifactStagingDirectory)/nupkg
```

Comment:
This previously tried to pack the template projects, resulting in a build pipeline error: `NETSDK1004: Assets file '/home/vsts/work/1/s/templates/UmbracoPackage/obj/project.assets.json' not found. Run a NuGet package restore to generate this file. [/home/vsts/work/1/s/templates/UmbracoPackage/UmbracoPackage.csproj`.

As mentioned in https://github.com/umbraco/Umbraco-CMS/pull/13481#discussion_r1117031652, setting the specific output path property for packages will also avoid the error introduced in .NET SDK 7.0.200 (see commit https://github.com/umbraco/Umbraco-CMS/commit/3ffe7f6d5da213be14269b31c11a541a783d94e1).